assets/js/components/user-input/util/constants.js

```diff
@@ -45,6 +45,16 @@ export const FORM_USER_INPUT_QUESTION_SNAPSHOT = 'user_input_question_snapshot';
 export const USER_INPUT_LEGACY_SITE_PURPOSE_DISMISSED_ITEM_KEY =
 	'user-input-legacy-site-purpose-dismissed-item';
 
+export const USER_INPUT_PURPOSE_TO_CONVERSION_EVENTS_MAPPING = {
```

Comment:
Following the existing pattern of storing user input related static constant, this is included in this file